### ISSUE TITLE
Keyboard controls not present in filter tabs

### ISSUE THREAD

**Issue by SEAjamieD** [chef/automate#3375]
## Describe the bug
Missing keyboard navigation in tab/card filtering based components listed in
applications/service-groups
compliance/reports/nodes
compliance/reports/profiles
settings/node-integrations/add

## To Reproduce
navigate to any/all of the URLs listed above
begin tabbing through the components, in each you will see a big chunk of the clickable elements that filter, are not keyboard accessible

## Expected behavior
When over the components outlined in red (in the screenshot) I should have a focus ring, and be able to click enter to select as filter

## Screenshots
![Screen Shot 2020-04-15 at 2.32.11 PM.png](https://images.zenhubusercontent.com/5d7aa5d5d4249c0001cab607/6a2c8829-1672-43a9-95bd-5b6a7520069d)![Screen Shot 2020-04-15 at 2.32.01 PM.png](https://images.zenhubusercontent.com/5d7aa5d5d4249c0001cab607/e35f6d6e-f0d5-4617-88b3-aaee9b404f24)

## Versions (please complete the following information):
- OS: Mac OS Mojave
- Browser:  chrome
